bridge-stp.in

```diff
@@ -83,6 +83,15 @@ MSTPD_ARGS=''
 MSTP_BRIDGES=''
 LOGGER='logger -t bridge-stp -s'
 
+# Set proper pidof command. Busybox doesn't support the -c argument, so test
+# for it and adjust our call accordingly
+pidof -c init
+if [[ $? -eq 0 ]] then
```

Comment:
double brackets [[ ]]  is valid mostly in bash
it doesn't work in classic /bin/sh

also, i would just use `pidof` without the `-c` argument

Reply:
Fair enough. That's done in the next commit.